### ISSUE TITLE
Pass events to inner components, like props.

### ISSUE THREAD

**Issue by muffeen** [matsp/material-components-vue#144]
**Is your feature request related to a problem? Please describe.**
On text field I wanted to pass a blur event to the input field, but it seems that the blur event is being passed to the root of the component instead.

**Describe the solution you'd like**
Just pass all events to input.

**Describe alternatives you've considered**
None.

**Additional context**
https://vuejs.org/v2/api/#vm-listeners, it would be good that like the `v-bind="$attr"`, `v-on="$listeners"` is also added.

**Comment by matsp** [comment]
@muffeen 
Is released..

**Comment by muffeen** [comment]
@matsp I'm not sure but on my end this seems not to work.
Here is a sample snippet I am working with.

```html
<template>
    <m-textfield v-model="input" @blur="test">
        ...
    </m-textfield>
</template>

<script>
    export default {
        methods: {
            test() {
                console.log('blur');
            }
        }
</script>
```

Blur event is not triggering. Is there something wrong on my code? Really big thanks for the help.

**Comment by matsp** [comment]
I will have a look later this day :innocent:

**Comment by matsp** [comment]
Should be fixed in 0.26.1 @muffeen
